### PR TITLE
Fixed blue LED bug with cayenne LPP device

### PR DESCRIPTION
**Pull request by simmoinard**
I had problems with my Things nodes (after several time, it freezes on fixed blue light). The problem has been solved on the forum (https://www.thethingsnetwork.org/forum/t/the-things-node-new-low-power-library-development/12521/144?u=simmoinard)
The "Basics.ino" file has been updated with a simple Delay(50), but not the "CayenneLPP.ino" ...

### CODE REVIEW

File: examples/CayenneLPP/CayenneLPP.ino [hunk 1]
```diff
@@ -119,6 +119,9 @@ void sendData(uint8_t port)
 
   // Set RN module to sleep mode
   ttn.sleep(60000);
+  // This one is not optionnal, remove it
+  // and say bye bye to RN2983 sleep mode
```
Comment (johanstokking) [inline examples/CayenneLPP/CayenneLPP.ino]: Please remove comment. While you're at it, also on line 101.

**Review by johanstokking — CHANGES_REQUESTED** [review]
Thanks, good catch. Two minor code style things.

**Comment by simmoinard** [comment]
Done !

**Review by johanstokking — APPROVED** [review]

**Comment by johanstokking** [comment]
Thanks!

**Comment by johanstokking** [comment]
Released in https://github.com/TheThingsNetwork/arduino-node-lib/releases/tag/v2.0.7

**Pull request merged**